lib/constants/error_messages.js

```diff
@@ -71,6 +71,7 @@ exports[404043] = 'Invalid clientConfigFile value. The specified value must be a
 exports[404044] = 'Invalid retryTimeout value. The specified value must be a number.';
 exports[404045] = 'Invalid account. The specified value must be a valid subdomain string.';
 exports[404046] = 'Invalid region. The specified value must be a valid subdomain string.';
+exports[404047] = 'Invalid clientStoreTemporaryCredential. The specified value must be a boolean.';
```

Comment:
Sounds weird to throw error on this and it makes more sense to me to fallback to external browser authentication. Have you checked the behavior of other drivers?

Reply:
Hmmm, I see. I am not sure other drivers have the same checking argument function. The reason why I put this error message is the node js driver usually creates an error message for each connection configuration. If you have a look at https://github.com/snowflakedb/snowflake-connector-nodejs/blob/master/lib/connection/connection_config.js#L214,  the driver has validated for each configuration and they have each error code.

Reply:
Ah I see. OK then.